### ISSUE TITLE
title attribute of logo link in header/footer makes no real sense

### ISSUE THREAD

**Issue by jackahl** [kitconcept/volto-light-theme#410]
Currently the title attribute of the Logo link in Header/Footer just says "Site". We should change this to the one in Plone Core: "Back to homepage"

**Comment by davisagli** [comment]
@jackahl No, this was discussed extensively and we arrived at the current solution. Let me try to find the link...

**Comment by jackahl** [comment]
@davisagli I had a read through that discussion in volto core and the conclusion was not what we have in VLT right now: https://github.com/kitconcept/volto-light-theme/pull/337

**Comment by davisagli** [comment]
@jackahl See https://github.com/kitconcept/volto-light-theme/pull/337 and the discussion on https://github.com/plone/volto/pull/5722 and https://github.com/plone/volto/issues/5720

**Comment by jackahl** [comment]
@davisagli oh, just noticed #337 is indeed in vlt. Just not deployed to https://light-theme.kitconcept.io/. Sorry for the noice. Closing

**Comment by davisagli** [comment]
@jackahl I'm pretty confused.

#337 was merged but is not correct -- see my comment there https://github.com/kitconcept/volto-light-theme/pull/337/files#r1624514733

Volto core currently does not set a title attribute or use the text "Back to homepage" (https://github.com/plone/volto/blob/main/packages/volto/src/components/theme/Logo/Logo.jsx#L50) so I'm confused why you are saying we should do that to match Volto core.

I agree volto-light-theme should do the same thing that is already in Volto core, I'm just not sure we are seeing the same thing for what that is.

**Comment by jackahl** [comment]
@davisagli nevermind. You are fully correct. I got confused with the whole thing. We will port the core implementation over to vlt at the sprint.
